### PR TITLE
authorized access points

### PR DESCRIPTION
**Pull request by rerowep**
* Closes https://github.com/rero/rero-ils/issues/2559
    * Takes the transliterated form instead of original script in
      authorised access point

### CODE REVIEW

**Review by PascalRepond — APPROVED** [review]

**Review by jma — COMMENTED** [review]
Changes you commit message -> closes

File: poetry.lock [hunk 1]
```diff
@@ -1039,9 +1039,9 @@ invenio-i18n = ">=1.2.0"
 [package.extras]
 all = ["Sphinx (>=3)", "cachelib (>=0.1)", "pytest-invenio (>=1.4.1)", "redis (>=2.10.5)"]
 docs = ["Sphinx (>=3)"]
-mysql = ["invenio-db[mysql,versioning] (>=1.0.8)"]
-postgresql = ["invenio-db[postgresql,versioning] (>=1.0.8)"]
-sqlite = ["invenio-db[sqlite,versioning] (>=1.0.8)"]
+mysql = ["invenio-db[versioning,mysql] (>=1.0.8)"]
```
Comment (jma) [inline poetry.lock:1042]: Add "update deps" in your commit message.

File: rero_mef/agents/tasks.py [hunk 2]
```diff
@@ -66,23 +65,17 @@ def task_create_mef_for_agent(pid, agent, dbcommit=True, reindex=True,
...
                 online=online
             )
-        mef_pid = 'Non'
-        if mef_record:
-            mef_pid = mef_record.pid
-        viaf_pid = 'Non'
-        if viaf_record:
-            viaf_pid = viaf_record.pid
-
+        mef_pid = mef_record.pid if mef_record else 'Non'
```
Comment (jma) [inline rero_mef/agents/tasks.py:75]: What `Non` means?

File: rero_mef/api.py [hunk 3]
```diff
@@ -513,15 +502,12 @@ def _index_action(self, payload):
...
             '_type': doc_type,
             '_id': str(record.id),
             '_version': record.revision_id,
             '_version_type': self._version_type,
             '_source': body
-        }
-        action.update(arguments)
-
-        return action
+        } | arguments
```
Comment (jma) [inline rero_mef/api.py:513]: I do not understand.

File: rero_mef/monitoring.py [hunk 4]
```diff
@@ -261,25 +257,24 @@ def missing_pids(doc_type):
...
-            else:
-                data['ES'].append(pid)
-        for pid in mon.get('ES duplicate'):
-            if api_url:
-                url = f'{api_url}?q=pid:{pid}'
-                data['ES duplicate'][url] = len(mon.get('ES duplicate'))
-            else:
-                data['ES duplicate'][pid] = len(mon.get('ES duplicate'))
-        return jsonify({'data': data})
+    data = {'DB': [], 'ES': [], 'ES duplicate': []}
```
Comment (jma) [inline rero_mef/monitoring.py:260]: Maybe you can mention this in the commit message.

**Review by jma — APPROVED** [review]

**Pull request merged**